jest-common/src/test/java/io/searchbox/cluster/TasksInformationTest.java

```diff
@@ -0,0 +1,15 @@
+package io.searchbox.cluster;
+
+import io.searchbox.action.Action;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class TasksInformationTest {
+
+    @Test
+    public void testUriGeneration() {
+        Action action = new TasksInformation.Builder("node_id:task_id").build();
```

Comment:
There is a confusion when it comes to what the taskId is - in fact the _tasks/<taskId> GET api actually expects to receive as the taskId  the key by which the task is stored in the result of the _tasks GET api (e.g. "oTUltX4IQMOUUVeiohTt8A:123"). That value is a string constructed from the node id (a string), a ":", and then the task id (which is an int).
I will rename the argument from taskId, hopefully it will help a bit...